gazebo/create_dockerfiles.py

```diff
@@ -24,19 +24,19 @@ def main(argv=sys.argv[1:]):
     parser.set()
     args = parser.parse(argv)
 
-    # Read platform perams
+    # Read platform params
```

Comment:
@ruffsl Maybe we should deduplicate these files as they seem to be identical between gazebo ros and ros2.
Do you expect these to diverge in the future or would it be safe to consolidate them ?

Reply:
Originally they were each slightly different, but after refactoring the case specific logic into docker_templates lib, they should be the same. I think the CI makes the assumption that the scripts are executed in place, but we could probably just move `create_dockerfiles.py` `create_dockerfolders.py` and `create_dockerlibrary.py` to the root of the repo (e.g. `/scripts`) and just symlink them to each `ros`, `ros2`, `gazebo` folder.